### DISCUSSION TITLE
无法修改mac地址

### DISCUSSION THREAD

**Discussion by oddhw** [Mygod/VPNHotspot#531] [Q&A] [answered]
![Screenshot_2024-01-19-08-48-57-93_593eae927fbf97881fe56889877096e4.jpg](https://github.com/Mygod/VPNHotspot/assets/71478520/5404a00d-b6c2-4cbe-9f38-e6ece8fdc93d)

不论我选择什么模式，mac地址在保存后还会变成原来的样子 我已经获取了root

**Comment by Mygod** [comment, marked as answer]
This dialog only suggests the configuration to Android system. It is up to the Android system to use or ignore the config, unfortunately.
